include/aos/iam/provisionmanager.hpp

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_PROVISIONMANAGER_HPP_
+#define AOS_PROVISIONMANAGER_HPP_
+
+#include <aos/common/tools/error.hpp>
```

Comment:
We treat aos/common headers as local inside aos_core_lib_cpp.

Reply:
Done